src/containerapp/azext_containerapp/custom.py

```diff
@@ -1650,6 +1747,28 @@ def patch_apply(cmd, resource_group_name=None, managed_env=None, show_all=False)
     patch_apply_handle_input(cmd, patchable_check_results, "y", pack_exec_path)
 
 
+def use_cloud_patch(cmd, container_app_name, resource_group_name, patch_name):
+    try:
+        app = show_containerapp(cmd, container_app_name, resource_group_name)
```

Comment:
Will this throw a not found exception if container app does not exist? Probably need to handle it separately? by check if not found exception is thrown in a separate method and return `None` if it is